### ISSUE TITLE
Changing the language on the article page leads to a non-existent page

### ISSUE THREAD

**Issue by trofimenk0** [strapi/LaunchPad#23]
## Trouble with language switcher

When I'm on the page of a single article and want to change the language, the browser goes to a non-existent page.
For example:
1. The current language is English. I am at the url - /en/blog/not-a-guide-to-integrating-strapi-with-next-js. 
2. I switch the language to French
3. I am on the page with url - /fr/blog/not-a-guide-to-integrating-strapi-with-next-js which is wrong webpage
![image](https://github.com/user-attachments/assets/eee4cb4e-4ced-4a44-9318-486ea35dbf82)

### Expectations
To get to the correct page (the French version of the article)

I think the problem is that the article slug for the English and French versions is different and this is not handled in any way.

**Comment by Mcastres** [comment]
Indeed! This logic was not implemented at all! I will fix it today. Stay tuned!

**Comment by vcoisne** [comment]
It was done by maxime and will be improved even more soon!
